test/net_http_test.rb

```diff
@@ -230,7 +231,8 @@ def test_use_custom_configuration_to_combine_endpoints_into_one_resource
     semian_config["development"]["nethttp_default"] = DEFAULT_SEMIAN_OPTIONS
     sample_env = "development"
 
-    semian_configuration_proc = proc do
+    semian_configuration_proc = proc do |host, port|
+      next nil if host == "127.0.0.1" && port == 8474 # # disable if toxiproxy
```

Comment:
This is definitely *one* of the problems, if there are even more than one. I can't reproduce any more errors though.